### PR TITLE
fix yahoo api breakage. issue #153

### PR DESCRIPTION
**Pull request by yoosefi**
works now :)

aside from the endpoint changing, there were a couple changes to the data structure as well.

also yahoo doesn't use the old http://weather.codes anymore. i updated the WOEID lookup button to open a working online database.

### CODE REVIEW

**Comment by yoosefi** [comment]
Aside from the endpoint changing, there were a couple changes to the data structure as well.

File: applet.js [hunk 1]
```diff
@@ -804,9 +792,9 @@ MyApplet.prototype = {
   }
 
 , weatherUrl: function weatherUrl() {
-    let output = QUERY_URL + ' where woeid="' + this._woeid + '" and u="' + this.unitToUrl() + '"'
+    //let output = QUERY_URL + ' where location="' + this._woeid + '" and u="' + this.unitToUrl() + '"'
     //let output = QUERY_URL + this._woeid + '_' + this.unitToUrl() + '.xml"'
-    return output
+    return 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20weather.forecast%20where%20woeid%20%3D%20%22' + this._woeid + '%22&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys'
```
Comment (mockturtl) [inline applet.js]: @yoosefi Did you find some documentation for this?  Where did the `env=store://datatables.org/alltableswithkeys` bit come from?

**Comment by yoosefi** [comment]
https://developer.yahoo.com/weather/

**Comment by mockturtl** [comment]
Hello,

This issue is being closed because development is moving to [linuxmint/cinnamon-spices/applets](https://github.com/linuxmint/cinnamon-spices-applets).  Feel free to reopen your issue there.

Please see the release announcement here:

http://segfault.linuxmint.com/2017/01/changes-to-cinnamon-spices-for-developer-and-artists/

**Pull request closed without merging**